src/entity/invoices/invoice.ts

```diff
@@ -156,6 +163,9 @@ export default class Invoice extends BaseEntity {
   })
   public date: Date;
 
+  @OneToMany(() => SubTransactionRow, (row) => row.invoice, { cascade: true })
+  public subTransactionRows: SubTransactionRow[];
+
```

Comment:
Wait... Cascade? So if the subtransactions are deleted, the invoice will be deleted too?